### DISCUSSION TITLE
Ubuntu automatic installation: server and...?

### DISCUSSION THREAD

**Discussion by onshisan** [automatic-ripping-machine/automatic-ripping-machine#552] [Q&A] [answered]
Reviewing the [wiki page for Ubuntu 20.04 Automatic Installation](https://github.com/automatic-ripping-machine/automatic-ripping-machine/wiki/Ubuntu-20.04-Automatic-Installation), I fouind the following:

> The ARM installation script supports both Ubuntu Server 20.04 and Ubuntu Server 20.04

Am I right that this should say Desktop and Server?

**Comment by 1337-server** [comment, marked as answer]
Yep!

Fixed now.
